Automattic/wp-calypso
# Remove current theme section.

This PR removes the "Current Theme Section" in Calypso,  seen below:

![screen shot 2018-04-03 at 6 13 42 pm](https://user-images.githubusercontent.com/5881557/38278627-d3bb3bb8-376a-11e8-94b4-c00fdcd69f63.png)

- - -

This has been proposed by @mapk in #23694.

The Current Theme UI is shown for both WPCOM sites, as well as Jetpack sites. Therefore, I would appreciate some help with the review from some of the following superstars,  to get this shipped:

Design: @mapk @kjellr 
Code: @ockham 
Jetpack Testing: @zinigor 
Extra Testing: @lancewillett 

- - -

### Test Instructions

1) Open _My Sites > Themes_ with both WPCOM and Jetpack sites.
2) Make sure everything works as expected without the _Current Theme UI_.

- - -

Thank you all for your help in advance, let's get this rolling!
